### PR TITLE
[Task][68326] create slide show with dummy data

### PR DESCRIPTION
**Pull request by TuanKieuDev**
### [Redmine link]
https://edu-redmine.sun-asterisk.vn/issues/68326

#### Screenshots (nếu có)

https://github.com/awesome-academy/ios_intern_kieu_anh_tuan-b_project_1/assets/56763942/0bba858b-9562-4c4b-aa19-20dcea7089aa

Screen Shot 2023-09-15 at 10 19 09

### CODE REVIEW

**Review by hlinh88 — APPROVED** [review]

File: MarvelApp/MarvelApp/Controller/Home/HomeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+//  ViewController.swift
+//  MarvelApp
+//
+//  Created by Tobi on 11/09/2023.
+//
+
+import UIKit
+
+final class HomeViewController: UIViewController {
+    @IBOutlet private weak var slides: UICollectionView!
```
Comment (tunghs-1253) [inline MarvelApp/MarvelApp/Controller/Home/HomeViewController.swift]:
```suggestion
    @IBOutlet private weak var slideCollectionView: UICollectionView!
```

File: MarvelApp/MarvelApp/Controller/Home/HomeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        slides.dataSource = self
+    }
+
+    private func startTimer() {
+        timer = Timer.scheduledTimer(timeInterval: 2.0,
+                                     target: self,
+                                     selector: #selector(timerAction), userInfo: nil, repeats: true)
+    }
+
+    @objc private func timerAction() {
```
Comment (tunghs-1253) [inline MarvelApp/MarvelApp/Controller/Home/HomeViewController.swift]: Tên func phải là động từ

File: MarvelApp/MarvelApp/Controller/TabBar/AppTabBarController.swift [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+//  Created by Tobi on 13/09/2023.
+//
+
+import UIKit
+
+final class AppTabBarController: UITabBarController {
+    private let home = HomeViewController()
+    private let search = SearchViewController()
+    private let comic = ComicViewController()
+    private let favorite = FavoriteViewController()
```
Comment (tunghs-1253) [inline MarvelApp/MarvelApp/Controller/TabBar/AppTabBarController.swift]:
```suggestion
    private let homeViewController = HomeViewController()
    private let searchViewController = SearchViewController()
    private let comicViewController = ComicViewController()
    private let favoriteViewController = FavoriteViewController()
```

File: MarvelApp/MarvelApp/View/Cell/SlideCollectionViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+final class SlideCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet private weak var images: UIImageView!
+
+    var image: UIImage! {
+        didSet {
+            images.image = image
+        }
+    }
```
Comment (tunghs-1253) [inline MarvelApp/MarvelApp/View/Cell/SlideCollectionViewCell.swift]: Không force unwrap

**Pull request merged**